### ISSUE TITLE
Register enhancement

### ISSUE THREAD

**Issue by am4nr** [flyth/ts3soundbot#98]
At the moment if you type in !register [username] you need an admin to give you a password and assign some rights. But i thougt it would be usefull if you type in !register [username] and then the bot message you "User created please send me your wanted password" so you can set the password for yourself. Then you get a standard account which an Admin could set up before with some rights like playback and upload

TL:DR let the user set their password while account creation

**Comment by Khailz** [comment]
I support that, I always forgot to make a topic on it though.

**Comment by flyth** [comment]
That has already been implemented but was buggy. Will be fixed in the next version.

**Comment by flyth** [comment]
Implemented as !password <password>.
